src/main/kotlin/no/nav/syfo/personoppgavehendelse/kafka/PersonoppgavehendelseService.kt

```diff
@@ -1,155 +0,0 @@
-package no.nav.syfo.personoppgavehendelse.kafka
-
-import no.nav.syfo.application.database.DatabaseInterface
-import no.nav.syfo.domain.PersonIdent
-import no.nav.syfo.kafka.KafkaConsumerService
-import no.nav.syfo.personstatus.db.*
-import no.nav.syfo.personstatus.domain.*
-import no.nav.syfo.util.callIdArgument
-import no.nav.syfo.util.kafkaCallId
-import org.apache.kafka.clients.consumer.ConsumerRecords
-import org.apache.kafka.clients.consumer.KafkaConsumer
-import org.slf4j.LoggerFactory
-import java.sql.Connection
-import java.sql.SQLException
-import java.time.Duration
-
-class PersonoppgavehendelseService(
-    private val database: DatabaseInterface,
-) : KafkaConsumerService<KPersonoppgavehendelse> {
-    override val pollDurationInMillis: Long = 100
-    private val isUbehandlet = true
-    private val isBehandlet = false
-
-    override fun pollAndProcessRecords(kafkaConsumer: KafkaConsumer<String, KPersonoppgavehendelse>) {
-        val records = kafkaConsumer.poll(Duration.ofMillis(pollDurationInMillis))
-
-        if (records.count() > 0) {
-            log.info("TRACE: Received ${records.count()} records")
-            processRecords(records)
-            kafkaConsumer.commitSync()
-        }
-    }
-
-    private fun processRecords(records: ConsumerRecords<String, KPersonoppgavehendelse>) {
-        // TODO: vurdere å legge inn støtte for tombstoned records
-
-        database.connection.use { connection ->
-            records.forEach { record ->
-                val callId = kafkaCallId()
-                val personoppgaveHendelse = record.value()
-                try {
-                    processPersonoppgavehendelse(
-                        connection,
-                        personoppgaveHendelse,
-                        callId,
-                    )
-                } catch (sqlException: SQLException) {
-                    // retry once before giving up (could be database concurrency conflict)
-                    log.info("Got sqlException, try again, callId: $callId")
-                    processPersonoppgavehendelse(
-                        connection,
-                        personoppgaveHendelse,
-                        callId,
-                    )
-                }
-                log.info("TRACE: Finished processing record! callId: $callId")
-                COUNT_KAFKA_CONSUMER_PERSONOPPGAVEHENDELSE_READ.increment()
-            }
-
-            connection.commit()
-        }
-    }
-
-    fun processPersonoppgavehendelse(
-        connection: Connection,
-        kPersonoppgavehendelse: KPersonoppgavehendelse,
-        callId: String,
-    ) {
-        val oversikthendelseType = kPersonoppgavehendelse.oversikthendelseType()
-        if (oversikthendelseType == null) {
-            handleMissingPersonoppgavehendelseType(
-                hendelsetype = kPersonoppgavehendelse.hendelsetype,
-                callId = callId,
-            )
-        } else {
```

Comment:
Denne funksjonaliteten er fjernet nå. Vi fant ikke helt ut hvorfor vi måtte sjekke for dette, når det er vi som konsumerer fra `ispersonoppgave` og da sender vi dette med derfra uansett (se https://github.com/navikt/ispersonoppgave/blob/4218288f2ccec529ecebfcf43d912e8cfffc8b7a/src/main/kotlin/no/nav/syfo/personoppgavehendelse/PersonoppgavehendelseProducer.kt#L20) 🤷🏼‍♂️ Sjekket også loggene og prometheus i prod, og var ingen forekomster av dette siste 60 dager.